src/selectors/entities/posts.js

```diff
@@ -230,7 +230,9 @@ export function makeGetPostsInChannel() {
                 }
 
                 const previousPost = allPosts[postIds[i + 1]] || {create_at: 0};
-                posts.push(formatPostInChannel(post, previousPost, i, allPosts, postsInThread, postIds, currentUser));
+                if (post) {
```

Comment:
I don't mind being defensive here at all, but out of curiosity why did you need it?

Reply:
Ah, i missed to revert this. This change is not needed anymore. (The problem I'm seeing initially is already fixed. It was fixed by adding `channel_id` in the combinedPost)